src/main/webapp/app/entities/exercise-hint/exercise-hint.model.ts

```diff
@@ -0,0 +1,5 @@
+import { Exercise } from 'app/entities/exercise';
+
+export class ExerciseHint {
+    constructor(public id?: number, public title?: string, public content?: string, public exercise?: Exercise) {}
```

Comment:
I suggest to write it like this:

    export class ExerciseHint implements BaseEntity {
        public title: string;
        public content: string;
        public exercise: Exercise;

        constructor() {
        }
    }

Because this is much easier to understand and the attributes are not hidden in the constructor.
This would also be consistent with the way how we define other model classes in TypeScript, see e.g. programming-exercise.model.ts

Reply:
You are right, I'll streamline it!